eng/testing/linker/trimmingTests.targets

```diff
@@ -92,15 +92,13 @@
 
   <Target Name="PublishTrimmedProjects"
           DependsOnTargets="GenerateProjects">
-    <PropertyGroup>
-      <TestPublishTrimmedCommand>"$(DotNetTool)"</TestPublishTrimmedCommand>
-      <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) build /t:PublishTrimmed</TestPublishTrimmedCommand>
-      <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) /nr:false</TestPublishTrimmedCommand>
-      <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) /warnaserror</TestPublishTrimmedCommand>
-      <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) -p:configuration=$(Configuration)</TestPublishTrimmedCommand>
-    </PropertyGroup>
+    <MSBuild Projects="@(TestConsoleApps)"
+             Targets="Restore"
+             Properties="MSBuildRestoreSessionId=$([System.Guid]::NewGuid());Configuration=$(Configuration);@(CliEnvironment)" />
```

Comment:
What is the MSBuildRestoreSessionId for?

Reply:
IIRC, this is for the static graph restore

Reply:
That doesn't have anything to do with static graph restore. It's a property that is set when restore is invoked via MSBuild so that conditions can be applied on that property to determine if the current node is used for restore:

https://github.com/dotnet/msbuild/blob/01ceb4e6dadb3fa41c90626899d68d640aa22db4/src/MSBuild/XMake.cs#L1420-L1423